api/grpcserver/interface.go

```diff
@@ -56,6 +56,14 @@ type atxProvider interface {
 	GetMalfeasanceProof(id types.NodeID) (*types.MalfeasanceProof, error)
 }
 
+type postState interface {
+	// PostStates returns the current state of all registered IDs.
+	// Meaning:
+	// * 0 - idle
+	// * 1 - proving.
```

Comment:
I would just refer to the type that implements the enum, otherwise if we change states we might forget to update the comment here.